tests/unit/models/PinsTest.php

```diff
@@ -49,4 +51,51 @@ public function testTogglePin(): void
         $this->assertTrue($this->Templates->Pins->isPinned());
         $this->assertTrue(count($this->Templates->Pins->readAll()) > 0);
     }
+
+    /**
+     * ensure that duplicated entry is not pinned
+     * @return void
+     */
+    public function testDuplicateIsNotPinned(): void
+    {
+        // duplicate existing experiment
+        $source = $this->Experiments;
+        $duplicate = $source->duplicate();
+        $freshSource = new Experiments($this->Users, $duplicate);
+        $this->assertNotEquals($source->entityData['id'], $freshSource->entityData['id']);
+
+        // ensure the duplicate experiment is not pinned
+        $this->assertCount(0, $freshSource->Pins->readAll());
+        $this->assertCount(0, $freshSource->Pins->readAllSimple());
+
+        // duplicate existing item
+        $sourceItem = $this->Items;
+        $duplicateItem = $sourceItem->duplicate();
+        $freshItem = new Items($this->Users, $duplicateItem);
+        $this->assertNotEquals($sourceItem->entityData['id'], $freshItem->entityData['id']);
+
+        // ensure the duplicate item is not pinned
+        $this->assertCount(0, $freshItem->Pins->readAll());
+        $this->assertCount(0, $freshItem->Pins->readAllSimple());
+    }
+
+    /**
+     * ensure that duplicating a pinned/non-pinned template results in a new pinned template in both cases
+     * @return void
+     */
+    public function testTemplateIsAlwaysPinnedWhenCreated(): void
+    {
+        // create a new template and ensure it is pinned
+        $source = new Templates($this->Users, 1);
+        $this->assertTrue($source->Pins->isPinned());
+        $this->assertTrue(count($source->Pins->readAll()) > 0);
+
+        // duplicate the template
+        $duplicate = $source->duplicate();
+        $fresh = new Templates($this->Users, $duplicate);
+        $this->assertNotEquals($source->entityData['id'], $fresh->entityData['id']);
+
+        // ensure the duplicate is pinned on creation
+        $this->assertTrue($fresh->Pins->isPinned());
```

Comment:
the function doing the duplication could return the fresh entity, so you can further assert differently on it for Templates.